code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -40,25 +40,7 @@ This file's folder contains:
 	return M && istype(M) && M.mind && ticker && ticker.mode && (M.mind in ticker.mode.servants_of_ratvar)
 
 /proc/is_eligible_servant(mob/living/M)
-	if(!istype(M))
-		return 0
-	if(!M.mind)
-		return 0
-	if(ishuman(M) && (M.mind.assigned_role in list("Captain", "Chaplain")))
```

Comment:
xhuisssssss i'm pretty sure the ishuman() was needed so that if you get your mind moved around you can be converted